spec/support/concerns/updatable.rb

```diff
@@ -0,0 +1,52 @@
+RSpec.shared_examples "it can be updated" do |schema|
```

Comment:
Could we update this test so that it accepts an example name that differs to the schema name as an optional argument?